docker-for-mac/index.md

```diff
@@ -35,9 +35,10 @@ channels, see the [FAQs](/docker-for-mac/faqs.md#stable-and-beta-channels).
     <th style="font-size: x-large; font-family: arial">Beta channel</th>
   </tr>
   <tr valign="top">
-    <td width="50%">This installer is fully baked and tested, and comes with the latest GA version of Docker Engine. <br><br>This is the best channel to use if you want a reliable platform to work with. <br><br>These releases follow a version schedule with a longer lead time than the betas, synched with Docker Engine releases and hotfixes.<br><br>On the stable channel, you can select whether to send usage statistics and other data.
+    <td width="50%">This installer is fully baked and tested, and comes with the latest GA version of Docker Engine along with an option to try <a href="https://github.com/docker/docker/tree/master/experimental">
+    experimental features</a>, which is configurable on Docker Daemon preferences for experimental mode. <br><br>This is the best channel to use if you want a reliable platform to work with. <br><br>These releases follow a version schedule with a longer lead time than the betas, synched with Docker Engine releases and hotfixes.<br><br>On the stable channel, you can select whether to send usage statistics and other data.
```

Comment:
@mstanleyjones I have a full topic on how to work with experimental mode w/screen snaps and detail, so I don't want to put that info in the "download installer" tables. I've updated both the Mac and Windows installer download descriptions, as well as experimental mode descriptions, which are linked to now from the installer tables.